src/index.ts

```diff
@@ -1,3 +1,14 @@
 export { SignedOrder } from '@0x/order-utils';
 export { serverRoutes } from './router';
-export { FirmQuote, RFQTFirmQuote, RFQMFirmQuote, TakerRequest, IndicativeQuote, RFQTIndicativeQuote, RFQMIndicativeQuote, Quoter, SubmitReceipt, SubmitRequest } from './types';
+export {
+    FirmQuote,
+    RFQTFirmQuote,
+    RFQMFirmQuote,
+    TakerRequest,
+    IndicativeQuote,
+    RFQTIndicativeQuote,
+    RFQMIndicativeQuote,
+    Quoter,
+    SubmitReceipt,
+    SubmitRequest,
+} from './types';
```

Comment:
Nothing changed here... right?  Did you mean to export the new `TakerRequestQueryParams` from here?  If not, you wouldn't be able to use that new type from the monorepo, right? (Admittedly, I haven't yet reviewed your monorepo PR to see whether/how you're using it from there.)

Reply:
Yeah! nothing changed here, I just ran the linted

Reply:
After reviewing your other (monorepo) PR, I'd like to ask that you consider exporting the new `TakerRequestQueryParams` type from here, so that you don't have to refer to it by package sub-path (`.../lib/src/...`) in the import of it into the monorepo.

Reply:
omg, that is so true! thanks for catching it!